### DISCUSSION TITLE
一款简约的 Obsidian 主题

### DISCUSSION THREAD

**Discussion by giscus** [loveminimal/comment#30] [General]
# 一款简约的 Obsidian 主题

一款简约的 Obsidian 主题 - https://aituyaa.com/%E4%B8%80%E6%AC%BE%E7%AE%80%E7%BA%A6%E7%9A%84-obsidian-%E4%B8%BB%E9%A2%98/

https://aituyaa.com/%E4%B8%80%E6%AC%BE%E7%AE%80%E7%BA%A6%E7%9A%84-obsidian-%E4%B8%BB%E9%A2%98/

**Comment by MarksonChen** [comment]
最喜欢的主题👏👏 辛苦了！！

**Reply by loveminimal** [reply]
^_^

**Comment by wjianbo** [comment]
感谢博主！很喜欢您做的这款主题，我想此基础加一些改动，做一款自己的主题上传可以吗？

**Reply by loveminimal** [reply]
可以 ~ 用着爽就行

**Comment by RinCat13** [comment]
你好，真的是非常好的主题，但是还是希望提供一些自定义选项。
例如：取消顶部的渐变条、
           可以隐藏滚动条、
           自定义背景颜色等。
谢谢！

**Reply by loveminimal** [reply]
近期应该没有时间去修改~ 
可以的话，fork 一份去魔改~
使用愉快🥳
